ara/callback/log_ara.py

```diff
@@ -0,0 +1,226 @@
+# This program is free software: you can redistribute it and/or modify
```

Comment:
Oh man, this must've been carried from human_log's header, this is nothing like human_log. Should switch this.